templates/zerver/dev_tools.html

```diff
@@ -0,0 +1,43 @@
+{% extends "zerver/portico.html" %}
+
+{# Login page. #}
+{% block portico_content %}
+<div id="devtools-page">
+    <h2>Useful development URLs</h2>
+    <p>
+        {% trans %}
+        Below is a list of useful URLs that provide information useful in development.
+        Most of these require you to run a command to build the relevant content. This
+        table specifies which command to use for each page. Make sure your development server
+        is still running when you visit these!
+        {% endtrans %}
+    </p>
+    <table class="table table-striped table-rounded table-bordered">
+        <thead>
+            <tr>
+                <th>Command</th>
+                <th>URL</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td><code>./tools/test-backend</code></td>
+                <td><a href="/coverage/index.html">/coverage/index.html</a></td>
+            </tr>
+            <tr>
+                <td><code>./tools/test-js-with-node --coverage</code></td>
+                <td><a href="/node-coverage/index.html">/node-coverage/index.html</a></td>
+            </tr>
+            <tr>
+                <td><code>cd docs &amp;&amp; make html</code></td>
```

Comment:
I think we might want to replace this with a tiny `tools/build-docs` wrapper, just for similarity with the rest.